vuepress/docs/next/tutorials/create/mfe/authentication.md

```diff
@@ -141,12 +137,13 @@ At `components/ConferenceDetailsContainer.js`
         </ThemeProvider>
       );
     }
-
+```
 > **Note**
 >
-> Keycloak object is accessible via props because of `withKeycloak` HOC:
+> A Keycloak object is accessible via props because of the `withKeycloak` HOC:
```

Comment:
properties? weird to abbreviate it

"because of" begs additional explanation; "due to," or reorder to "The `withKeycloak` HOC allows properties to access the Keycloak object." i realize you may want to lead with the keycloak object, but reversing the order avoids the awkward wording

i think "A Keycloak" -> "The Keycloak" ?? does this high order calculator apply to all keycloak objects or this one in particular? if it applies in general it's probably better not to call out that detail (we limit discussion to our own product without delving into how others work unless somehow relevant)

Reply:
props is a value in the code

edited.

@nshaw, is this HOC High order component or high order calculator?

Reply:
High Order Component